pal/screen-adapter/native/screen-adapter.ts

```diff
@@ -28,9 +28,13 @@ class ScreenAdapter extends EventTarget {
 
     public get windowSize (): Size {
         const dpr = this.devicePixelRatio;
+        // @ts-expect-error interface for OH only
+        const width = globalThis.oh ? jsb.device.getInnerWidth() : window.innerWidth;
```

Comment:
Just use `globalThis.oh` to distinguish oh platform? Does it have more suitable way, such as sys.oh.

Reply:
The currently used window.oh, here is the old implementation

Reply:
Why not use sys.oh?

Reply:
`window.oh` is not a boolean value, it's an **engine-internal** global object that contains the TS system ability that we use in engine, for example `window.oh.display` to query the screen size